src/components/technologies/genericTechnologies/conv1.py

```diff
@@ -838,25 +839,73 @@ def bind_disjunctions(dis, t):
             )
 
         else:
+            if data["config"]["optimization"]["typicaldays"]["N"]["value"] == 0:
+                input_aux = self.input
+                set_t = self.set_t_performance
+            else:
+                if (
+                    data["config"]["optimization"]["typicaldays"]["method"]["value"]
+                    == 1
+                ):
+                    sequence = data["k_means_specs"]["sequence"]
+                elif (
+                    data["config"]["optimization"]["typicaldays"]["method"]["value"]
+                    == 2
+                ):
+                    sequence = self.sequence
+
+                # init bounds at full res
+                bounds_RR_full = {
```

Comment:
please keep variable names lower case (i.e. bounds_rr_full). I know it doesnt look super beautiful, but thats just what it is... :D

Reply:
(holds for everything else as well where we have a RR)

Reply:
done